tests/plugins/widget/manual/keystroke.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: widget, bug, 4.14.1, 3998
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, sourcearea
+
+### Scenario 1
+
+1. Open console in dev tools.
+
+1. Place cursor inside editor.
+
+1. Press CTRL and Enter key.
+
+#### Expected
+
+Source mode is opened and no error appears in dev console.
+
+#### Unexpected
+
+Error appears in dev console or source code editor is not opened.
```

Comment:
```suggestion
Error appears in dev console or source mode editor is not opened.
```

Remember to be consistent when it comes to naming - it makes understanding test scenarios much easier. Also it's in fact called `source mode` so this name should be used.